cluster-support-bot.py

```diff
@@ -142,40 +142,51 @@ def handle_help(payload, args=None, body=None, subparser=None):
     message = subparser.format_help()
     return web_client.chat_postMessage(channel=channel, thread_ts=thread, text=message)
 
+def _block_from_text(line):
+    return {
+        "type": "section",
+        "text": {
+            "type": "mrkdwn",
+            "text": line
+        }
+    }
 
 def handle_summary(payload, args=None, body=None):
     web_client = payload['web_client']
     channel = payload['data']['channel']
     thread = payload['data'].get('thread_ts', payload['data']['ts'])
     cluster = args.cluster
+    blocks = []
     try:
         summary, _ = get_summary(cluster=cluster)
+        for line in summary:
+            blocks.append(_block_from_text(line))
     except ValueError as error:
         return web_client.chat_postMessage(
             channel=channel,
             thread_ts=thread,
             text='{} {}'.format(cluster, error))
-    return web_client.chat_postMessage(channel=channel, thread_ts=thread, text=summary)
-
```

Comment:
You want to keep this blank line for PEP 8's [two lines between top-level functions][1].

[1]: https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
done